kai.yaml

```diff
@@ -29,7 +29,7 @@ kubeconfig:
     token:
 
 # Which namespaces to search or exclude.
-namespaces:
+namespace-selectors:
```

Comment:
So this change is just to disambiguate them?

Reply:
Yes, the old `namespaces` config expects a slice while the new `namespaces` expects a map. So they need to be disambiguated since the underlying structure is different